packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/index.js

```diff
@@ -73,11 +76,22 @@ const TransferTokenCreateView = () => {
     },
     {
       enabled: !isCreating && !transferToken,
-      onError() {
-        toggleNotification({
-          type: 'warning',
-          message: { id: 'notification.error', defaultMessage: 'An error occured' },
-        });
+      onError(err) {
+        if (err?.response?.data?.error?.details?.code === 'INVALID_TOKEN_SALT') {
+          toggleNotification({
+            type: 'warning',
+            message: {
+              id: 'notification.error.invalid.configuration',
+              defaultMessage:
+                'You have an invalid configuration, check your server log for more information.',
+            },
+          });
+        } else {
+          toggleNotification({
+            type: 'warning',
+            message: { id: 'notification.error', defaultMessage: 'An error occured' },
+          });
```

Comment:
As we've discussed that yesterday: would it make sense to throw the actual error here to give users a chance to understand what happened?

Reply:
ok yes, done